org.virt_manger.virt-viewer.json

```diff
@@ -0,0 +1,321 @@
+        },
+        {
+            "name": "libnl",
+            "config-opts": [
+                "--enable-cli=no"
+            ],
+            "cleanup": [
+                "/etc"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/thom311/libnl/releases/download/libnl3_4_0/libnl-3.4.0.tar.gz",
+                    "sha256": "b7287637ae71c6db6f89e1422c995f0407ff2fe50cecd61a312b6a9b0921f5bf"
+                }
+            ]
+        },
+        {
+            "name": "gtk-vnc",
+            "cleanup": [
+                "/bin"
+            ],
+            "sources":[
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gtk-vnc/0.7/gtk-vnc-0.7.2.tar.xz",
+                    "sha256": "f893f32b1ef6d09dd23cda39b8a2567be7c2aebda58026288e1362f042e20808"
+                }
+            ]
+        },
+        {
+            "name": "spice-protocol",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://anongit.freedesktop.org/git/spice/spice-protocol.git",
+                    "tag": "v0.12.13",
+                    "commit": "8dda82b49d8f848a25e3a1ef6df943276c59e462"
+                }
+            ]
+        },
+        {
+            "name": "phodav",
+            "cleanup": [
+                "/bin"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnome.org/pub/GNOME/sources/phodav/2.2/phodav-2.2.tar.xz",
+                    "sha256": "080305dc9e804562f200ffc8c7d345c76754fce9cb01489dddb1b2c52c0557c1"
+                }
+            ]
+        },
+        {
+            "name": "libvirt",
+            "cleanup": [
+                "/bin",
+                "/share",
+                "/libexec",
+                "/etc"
+            ],
+            "config-opts": [
+                "--prefix=/app",
+                "--sysconfdir=/etc",
+                "--localstatedir=/var",
+                "--without-html-dir",
+                "--without-bash-completion",
+                "--without-test",
+                "--without-html-subdir",
+                "--without-storage-dir",
+                "--without-storage-fs",
+                "--without-storage-lvm",
+                "--without-storage-iscsi",
+                "--without-storage-scsi",
+                "--without-storage-mpath",
+                "--without-storage-disk",
+                "--without-storage-rbd",
+                "--without-storage-sheepdog",
+                "--without-storage-gluster",
+                "--without-storage-zfs",
+                "--without-storage-vstorage"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://libvirt.org/sources/libvirt-4.3.0.tar.xz",
```

Comment:
Not sure if it matters for app functionality, but 4.4.0 is o ut.

Reply:
Updated it (and a few other things) anyway.